apps/desktop-wallet/src/pages/UnlockedWallet/AddressesPage/AddressesTabContent.tsx

```diff
@@ -80,9 +67,7 @@ const AddressesTabContent = () => {
     >
       <TableGrid>
         <TableGridContent>
-          {visibleAddresses.map((address) => (
-            <AddressGridRow addressHash={address.hash} key={address.hash} />
-          ))}
+          {visibleAddresses?.map((addressHash) => <AddressGridRow addressHash={addressHash} key={addressHash} />)}
```

Comment:
What about adding a placeholder text here if no address is found?

Reply:
💯

Reply:
Done 
<img width="1628" alt="image" src="https://github.com/user-attachments/assets/4315702f-5d82-4aac-a895-6124ffe94024">